### ISSUE TITLE
Ecotone Tracker

### ISSUE THREAD

**Issue by merklefruit** [a16z/magi#200]
## Ecotone upgrade changes overview (WIP)

- [x] support blobs retrieval in the L1-traversal stage ([specs](https://github.com/ethereum-optimism/specs/blob/main/specs/derivation.md#ecotone-blob-retrieval))
  - this will require an L1 CL connection, which currently Magi doesn't have as it's not needed. 
  - for each batcher transaction of type `0x03` in a given block, we need to fetch blobs via the `/eth/v1/beacon/blob_sidecars/` endpoint, with the `indices` filter to skip unrelated blobs. 
  - blobs retrieved should be validated (but this can be skipped if the L1 CL is trusted)
  - blob transactions calldata needs to be ignored (not used in blobs version 0)
  - blobs need to be decoded
- [x] support ecotone L1 block info ([op-node ref](https://github.com/ethereum-optimism/optimism/pull/8786/files#diff-83684a8056b80b8473890eafdbcc2d0bf3e9909f3d0f2309ba0b54b4351aa0c5))
  - after Ecotone, the L1 Attributes transaction call changes from `setL1BlockValues` to `setL1BlockValuesEcotone`. 
  - in the Ecotone activation block (except if it's not active since genesis) the [ecotone upgrade transactions](https://github.com/ethereum-optimism/specs/blob/main/specs/derivation.md#network-upgrade-automation-transactions) need to be handled with a special case.
  - TL;DR on `setL1BlockValuesEcotone`: The new input args are no longer ABI encoded function parameters, but are instead packed into 5 32-byte aligned segments (starting after the function selector).
  - other minor changes: e.g. `basefeeScalar` and `blobBasefeeScalar ` go from u256 to u32.
- [x] add parent beacon block root
- [x] p2p gossip: added blocks/V3 topic
